test_utils/test_utils/paho_fuzz_hook.py

```diff
@@ -0,0 +1,249 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from . import logging_hook
+import random
+import functools
+
+
```

Comment:
As we discussed, I'm not sure this belongs in test_utils as opposed to the more specific fuzzing tests in sdklab, unless you see this being useful in other contexts as well